notebooks/recommendation_systems/solutions/tfrs_basic_retrieval.ipynb

```diff
@@ -0,0 +1,1364 @@
+{
```

Comment:
<ul><li>Does the "compute_loss" function required typing? If not, I suggest removing it. Same in the cell below.</li><li>Why is there a "positive" in "positive_movie_embeddings"? What does that mean?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<ul><li>Removed typing</li><li>In terms of the positive, I added an explanation in the <code>Metrics</code> section. </li></ul>